homeassistant/components/script.py

```diff
@@ -170,8 +172,15 @@ def is_on(self):
 
     async def async_turn_on(self, **kwargs):
         """Turn the script on."""
+        context = kwargs.get('context')
+        self.async_set_context(context)
+        self.hass.bus.async_fire(EVENT_SCRIPT_RUN, {
```

Comment:
So we have a bit of a weird thing right now with scripts that include a delay that get turned on . It means that we don't always start a new script, but sometimes just continue it. (it's kinda weird honestly). So we might want to check that.

Reply:
I will add an attribute to the event data to determine if it is a new run or a continuation based on self.script.is_running in the component, then add it to logbook for "{{name}} started" and "{{name}} continued" for scripts.  I currently only added automations to logbook.

Reply:
Looks like scripts running again to bypass delay has been disabled based on this: https://github.com/home-assistant/home-assistant/blob/dev/homeassistant/components/script.py#L118-L119

Reply:
Okay, then that's not a problem then !